torch/testing/_internal/common_device_type.py

```diff
@@ -788,6 +788,63 @@ def test_wrapper(*args, **kwargs):
                     print("Failed to instantiate {0} for op {1}!".format(test_name, op.name))
                     raise ex
 
+# Decorator that generates two set of tests with kwarg sparse_csr=True and sparse_csr=False.
+# All keyword arguments are used only if sparse_csr==True, so they have no effect on the original test
+class enableSparseCSR(_TestParametrizer):
```

Comment:
@pmeier, could you please review this PR, particularly `enableSparseCSR` functionality? This adds a simple `sparse_csr=True/False` and a bit of specific customization when `sparse_csr=True`.